_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -763,372 +763,2409 @@ Terraform::
+=== Configure the security baseline for the root account
+
+Next, we'll configure a security baseline for the root account that is responsible for creating all the child accounts.
+It will also configure AWS Organizations, IAM Roles, IAM Users, IAM Groups, IAM Password Policies, Amazon GuardDuty, AWS CloudTrail, AWS Config and Security Hub.
+
+We'll be using the `landingzone/account-baseline-root` module from https://github.com/gruntwork-io/terraform-aws-cis-service-catalog[terraform-aws-cis-service-catalog].
+
+[.exceptional]
+IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork subscriber# to access `terraform-aws-service-catalog`.
+
+First, create a _wrapper module_ called `account-baseline-root` in your `infrastructure-modules` repo under the `landingzone` subdirectory:
+
+----
+infrastructure-modules
+  └ landingzone
+    └ account-baseline-root
+      └ main.tf
+      └ outputs.tf
+      └ variables.tf
+----
+
+Inside of `main.tf`, configure your AWS provider and Terraform settings:
+
+.infrastructure-modules/landingzone/account-baseline-root/main.tf
+[source,hcl]
+----
+provider "aws" {
+  # The AWS region in which all resources will be created
+  region = var.aws_region
+
+  # Require a 3.x version of the AWS provider
+  version = "~> 3.40"
+
+  # Only these AWS Account IDs may be operated on by this template
+  allowed_account_ids = [var.aws_account_id]
+}
+
+terraform {
+  # The configuration for this backend will be filled in by Terragrunt or via a backend.hcl file. See
+  # https://www.terraform.io/docs/backends/config.html#partial-configuration
+  backend "s3" {}
+
+  # Only allow this Terraform version. Note that if you upgrade to a newer version, Terraform won't allow you to use an
+  # older version, so when you upgrade, you should upgrade everyone on your team and your CI servers all at once.
+  required_version = "= 0.12.26"
+}
+----
+
+Next, use the `account-baseline-root` module from the Gruntwork Infrastructure as Code Library:
+
+.infrastructure-modules/landingzone/account-baseline-root/main.tf
+[source,hcl]
+----
+module "root_baseline" {
+  source = "git::git@github.com:gruntwork-io/terraform-aws-cis-service-catalog.git//modules/landingzone/account-baseline-root?ref=v0.44.10"
+
+  aws_account_id = var.aws_account_id
+  aws_region     = var.aws_region
+  name_prefix    = var.name_prefix
+
+  # If you're running the example against an account that doesn't have AWS Organization created, change the following value to true
+  create_organization = var.create_organization
+
+  # The child accounts to create
+  child_accounts = var.child_accounts
+
+  # IAM users to create in this account
+  users = var.users
+
+  # These are variables you only need to set at test time so that everything can be deleted cleanly.
+  # You will likely NOT need to set this in any real environments.
+  force_destroy_users      = var.force_destroy
+  cloudtrail_force_destroy = var.force_destroy
+  config_force_destroy     = var.force_destroy
```

Comment:
Should we be mentioning them here then, considering this is a production guide?

Reply:
I think you might be right and we don't want these here!